components/nexus-blobstore-file/src/main/java/org/sonatype/nexus/blobstore/file/MapdbBlobMetadataStore.java

```diff
@@ -448,19 +460,28 @@ public long getBlobSize() {
     ensureStarted();
     return database.execute(new Fun.Function1<Long, DB>()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this anonymous class number of lines from 25 to at most 20, or make it a named class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1188)